host/xtest/regression_4000.c

```diff
@@ -5299,6 +5299,7 @@ static void xtest_tee_test_4012(ADBG_Case_t *c)
 }
 ADBG_CASE_DEFINE(regression, 4012, xtest_tee_test_4012,
 		"Test seeding RNG entropy");
+#endif /*CFG_SYSTEM_PTA*/
 
 static void xtest_tee_test_4013(ADBG_Case_t *c)
```

Comment:
This one uses `TA_CRYPT_CMD_DERIVE_TA_UNIQUE_KEY{,SHM}` which depend on `CFG_SYSTEM_PTA`